docs/docs/setup.md

```diff
@@ -1,43 +1,44 @@
 ---
 id: setup
-title: Setup
+title: Install and set up
 ---
 
-The following steps will get you started using ZeppelinOS.
+The following steps will get us started using ZeppelinOS.
 
 ## Installation
 
-`zos` is the command line interface to ZeppelinOS. To install it, run:
+`zos` is the [command line interface](https://github.com/zeppelinos/zos-cli) to ZeppelinOS. To install it, let's go to our terminal and run:
```

Comment:
I wouldn't put this link here, we don't want people to loose focus at this point, it is *here* you should be looking at, not the repo.

Reply:
Disagree, I don't find this link misleading, on the contrary, it's important to provide links for further exploration.